org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
In the alljoyn case, wouldn't you end up doing a call for each property of each device (at least how the translators are currently implemented)?

In that case, the number of calls seems to add up very quickly.

Is that not an expected scenario for the alljoyn protocol (enumerating devices and displaying the current state of each of them)?

---
In reply to: [76344264](https://github.com/openT2T/translators/pull/75#discussion_r76344264) [](ancestors = 76344264,76336787,76335569,76334693,76154047)